tests/integration/targets/openssh_cert/tests/key_idempotency.yml

```diff
@@ -63,11 +63,13 @@
         valid_from: always
         valid_to: forever
       register: second_signature_algorithm
+      when: openssh_version is version("8.7", "!=")
```

Comment:
This is likely the wrong check, since the tests pass with OpenSSH 9.0 on Arch Linux: https://dev.azure.com/ansible/community.crypto/_build/results?buildId=42954&view=logs&j=57f52a93-2f8f-5f4a-64e7-e08c30d14461&t=639a25f5-5b1b-5d56-c79d-060a9682ed36&l=312

Reply:
How about checking for RHEL 9 instead?

Reply:
Another datapoint: the tests pass with OpenSSH 8.7 on Fedora 35: https://dev.azure.com/ansible/community.crypto/_build/results?buildId=42954&view=logs&j=ad47e826-3d51-5052-c5d1-abf94a7b75a5&t=a7141e3f-34f2-5aff-b04a-ec77c032fc27&l=351

So it's really not 8.7 in general, but something that RHEL 9 does.

Reply:
Ah, not sure if you all have access to [this](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/considerations_in_adopting_rhel_9/assembly_security_considerations-in-adopting-rhel-9#ref_considerations-security-crypto_changes-to-security), but RHEL9 disables SHA1 in the default crypto-policies and would have to be explicitly enabled. I'll push the updated condition in a moment.

Reply:
Thanks! That document seems to be publicly visible.